R/error-checks.R

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think `error-checks.R` can be deleted. It's an empty file anyway.

Reply:
Yes but let's keep it for the consistency with other osps packages. So in case we will require a check function we know where to put it :)